lin_actuator.h

```diff
@@ -1,27 +1,37 @@
 /* 
```

Comment:
This header comment should just go. We usually have multiple authors per file and git keeps track of that much better. Not a must-fix but I'll remove this once merged.

Reply:
oops, i think mplab made it for some reason